### ISSUE TITLE
Removing files and/or folders by pressing Del key sometimes doesn't work

### ISSUE THREAD

**Issue by dvdesolve** [linuxmint/nemo#2481]
```
 * Version 4.6.4
 * Occurs in fullscreen and in window mode
 * Arch Linux
 * Integrated Intel with both open-source Intel and modesetting drivers; NVIDIA GTX 2070 SUPER with proprietary NVIDIA driver
 * 64 bit
 ```

**Issue**
Sometimes when I want to delete file or directory (or a selection of them) by pressing Del on my keyboard nothing happens - no errors, no actions, objects and selection remains the same. Few consecutive Del presses or re-selecting objects of interest with further pressing Del key solves the problem. Can't catch any regularity - it doesn't depend on file system, whether directory is full or not etc.

**Steps to reproduce**
Open Nemo, select file(s) and/or folder(s). Try to delete them by pressing Del key.

**Expected behaviour**
Objects will be deleted

**Comment by Jeremy7701** [comment]
Could it be that you have a bit of gunge under your Del key?
Do you get this problem when using the Del key in other areas - such as a CLI screen?
Have you tried a keyboard swap?

**Comment by dvdesolve** [comment]
no, it's definitely not the problem with dirty keyboard. in other apps it's working just fine. also I observe this issue on a bunch of another systems with the same setup

**Comment by SomeTroglodyte** [comment]
I started seeing this too after upgrading mint 19 to 20, imho definitely a regression. Maybe related to some kind of anti-bounce protection which nemo obviously has intentionally? Click something and hit F2 within half a second -> always ignored. This, though, still happens even after waiting several seconds.

**Comment by Joe1962** [comment]
This happens when you are inside a folder, then go up one and the folder is, of course, still selected, but Del won't work until you select the folder again. Interestingly, F2 always works, so I figured it might be some kind of built-in protection mechanism. But it does get annoying at times when you want to check inside a folder just before deleting it

**Comment by Joe1962** [comment]
Hmm... can't edit my comment.

I am currently on Mint 20, nemo 4.6.5, but have seen this since Mint 19.x, IIRC.

**Comment by icarter09** [comment]
Was anyone able to come up with a set of steps to reproduce this issue? Thanks.

**Comment by dvdesolve** [comment]
I'm observing it every day - like was described in the first post

**Comment by icarter09** [comment]
Nemo 4.8.4
Mint 20.1 64bit

Following the steps in the first post, I'm not able to reproduce this issue.

**Comment by icarter09** [comment]
@dvdesolve can you attach the file generated from `dconf dump /org/nemo/ > my_nemo_settings.txt` ? Thanks.

**Comment by dvdesolve** [comment]
@icarter09 here it is
```
[desktop]
computer-icon-visible=false
desktop-layout='false::false'
font='DejaVu Sans 9'
home-icon-visible=false
trash-icon-visible=false
volumes-visible=false

[preferences]
context-menus-show-all-actions=false
executable-text-activation='display'
never-queue-file-ops=true
show-full-path-titles=false
show-hidden-files=false
show-location-entry=true
thumbnail-limit=uint64 104857600

[window-state]
geometry='905x856+178+59'
maximized=true
my-computer-expanded=true
network-expanded=true
sidebar-bookmark-breakpoint=1
sidebar-width=219
start-with-sidebar=true
```

**Comment by icarter09** [comment]
@dvdesolve I'll give an update if I encounter the issue. Thanks.

**Comment by gdv1987** [comment]
I am having the same issue. 
Linux Mint 21.1 Cinnamon, clean install.
"dconf dump /org/nemo/"  attached to this message.

[my_nemo_settings.txt](https://github.com/linuxmint/nemo/files/10941568/my_nemo_settings.txt)

**Comment by Joe1962** [comment]
Ok, so I have recently been seeing a similar effect on one PC with Mint 20.3 (will try to confirm next week on 21.X) where if a file is already selected when returning to nemo from another software, then you Ctrl-select another one, then press Del, nothing happens. You have to re-select them both to delete them. Hope this can shed some light on where the ongoing issue lies.

**Comment by Jeremy7701** [comment]
I tried a `dconf dump /org/nemo/ >nemo-settings.txt` and got 100+  lines of settings:-
However, I have **no** issues - whatever - with nemo.

[nemo-settings.txt](https://github.com/linuxmint/nemo/files/10950801/nemo-settings.txt)

**Comment by CarlosRuiz-globalqss** [comment]
Confirmed, same happens to me sometimes, but not always.

It was happening after opened nemo coming from firefox.  I closed it and open the app directly and it worked fine again.

**Comment by logon84** [comment]
> Confirmed, same happens to me sometimes, but not always.
> 
> It was happening after opened nemo coming from firefox. I closed it and open the app directly and it worked fine again.

Same here. I opened the "Music" folder from Shortwave (radio streams program) and I coudn't delete any file using the DEL key. After reading your reply, I just closed the nemo window, opened it again and I could finally delete files using the DEL key. Using Mint 21.3 here.

**Comment by sophie-la-li** [comment]
Same issue on Mint 22.1. Closing and re-opening Nemo fixed it here, too.

**Comment by skrabadiloseidon** [comment]
Same issue on Mint 22.3. Sometimes DEL key just stops working.
